pkg/conf/conf.go

```diff
@@ -444,6 +459,7 @@ type Exporter struct {
 	Token         *string `yaml:"token,omitempty"`
 	Precision     *string `yaml:"precision,omitempty"`
 	ClientTimeout *string `yaml:"client_timeout,omitempty"`
+	Version       *string `yaml:"version,omitempty"`
```

Comment:
Nice catch! Make sure you include `version` in one of the tests too.

Reply:
Added the test.